02-apply_maxwell_filter.py

```diff
@@ -2,24 +2,24 @@
 ===================================
 03. Maxwell filter using MNE-python
 ===================================
-
-XXX: this needs update: congig.mf_reference_run, config.mf_st_duration
-config.mf_cal_fname, config.mf_ctc_fname
-
-The data are Maxwell filtered using tSSS and movement compensation.
-
+  
+The data are Maxwell filtered using SSS or tSSS (if config.mf_st_duration is not None)
+and movement compensation.
 Using tSSS with a short duration can be used as an alternative to highpass
 filtering. Here we will use the default (10 sec) and a short window (1 sec).
 
+The head position of all runs is corrected to the run specified in 
+config.mf_reference_run. 
+
 It is critical to mark bad channels before Maxwell
-filtering. Here for consistency we exploit the MaxFilter log files for
-determining the bad channels.
+filtering. 
+
+The function loads machine-specific calibration files from the paths set for
+config.mf_ctc_fname  and config.mf_cal_fname. 
 
-The data are also lowpass filtered at 40 Hz using linear-phase FIR filter with
-delay compensation. The transition bandwidth is automatically defined. See
-`Background information on filtering <http://mne-tools.github.io/dev/auto_tutorials/plot_background_filtering.html>`_
-for more. The filtered data are saved to separate files to the subject's'MEG'
-directory.
+# XXX – do we?
+Here for consistency we exploit the MaxFilter log files for
+determining the bad channels.
```

Comment:
do we don't as we don't have this information unless we did run maxfilter from elekta before.